tutorials/creating_sequences.ipynb

```diff
@@ -38,11 +41,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As presented in [\"Programming a neutral-atom QPU\"](../programming.md), Pulser enables you to program [an Hamiltonian](../programming.md) composed of an [interaction Hamiltonian](../programming.md) and a [drive Hamiltonian](../programming.md).\n",
+    "As presented in [\"Programming a neutral-atom QPU\"](../programming.md), Pulser enables you to program [an Hamiltonian](../programming.md#2-hamiltonian-evolves-the-state) composed of an [interaction Hamiltonian](../programming.md#22-interaction-hamiltonian) and a [drive Hamiltonian](../programming.md#21-driving-hamiltonian).\n",
```

Comment:
You have to leave out the numbers for this to work
```suggestion
    "As presented in [\"Programming a neutral-atom QPU\"](../programming.md), Pulser enables you to program [an Hamiltonian](../programming.md#hamiltonian-evolves-the-state) composed of an [interaction Hamiltonian](../programming.md#interaction-hamiltonian) and a [drive Hamiltonian](../programming.md#driving-hamiltonian).\n",
```
This applies everywhere in the notebook